clients/pkg/promtail/targets/heroku/target.go

```diff
@@ -117,6 +117,12 @@ func (h *Target) drain(w http.ResponseWriter, r *http.Request) {
 			ts = message.Timestamp
 		}
 
+		// Create __heroku_drain_param_<name> labels from query parameters
+		params := r.URL.Query()
+		for k, v := range params {
+			lb.Set(fmt.Sprintf("__heroku_drain_param_%s", k), strings.Join(v, ","))
```

Comment:
Is there a reason to prefix the keys? would it be better for users to just use the url param key as the label name?

Reply:
oh I see, I think the intent is to use a relabel rule to turn these into actual labels sent to Loki?

That seems reasonable to me 👍